src/main/java/fr/formation/user/User.java

```diff
@@ -11,76 +13,137 @@
 @Table(name = "user")
 public class User {
 
-	@Id
-	@GeneratedValue(strategy = GenerationType.AUTO)
-	private Long id;
-
-	@Column(name = "username")
-	private String username;
-
-
-	@Column(name = "password")
-	@JsonIgnore
-	private String password;
-
-
-	/**
-	 * Gets id.
-	 *
-	 * @return the id
-	 */
-	public Long getId() {
-		return id;
-	}
-
-
-	/**
-	 * Sets id.
-	 *
-	 * @param id the id
-	 */
-	public void setId(Long id) {
-		this.id = id;
-	}
-
-
-	/**
-	 * Gets username.
-	 *
-	 * @return the username
-	 */
-	public String getUsername() {
-		return username;
-	}
-
-
-	/**
-	 * Sets username.
-	 *
-	 * @param username the username
-	 */
-	public void setUsername(String username) {
-		this.username = username;
-	}
-
-
-	/**
-	 * Gets password.
-	 *
-	 * @return the password
-	 */
-	public String getPassword() {
-		return password;
-	}
-
-
-	/**
-	 * Sets password.
-	 *
-	 * @param password the password
-	 */
-	public void setPassword(String password) {
-		this.password = password;
-	}
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
 
+    @NotEmpty
```

Comment:
Est-ce que le NotEmpty crée la colonne avec ne champ not nullable en base ? Je ne pense pas